app/ami.py

```diff
@@ -0,0 +1,158 @@
+from dataclasses import dataclass
+
+from app import resource
+from .resource import Resource
+import boto3
+
+from datetime import datetime
+TODAY = datetime.today()
+TODAY_IS_WEEKEND = TODAY.weekday() >= 4  # Days are 0-6. 4=Friday, 5=Saturday, 6=Sunday, 0=Monday
+
+
+@dataclass
+class Ami(Resource):
+    state: str
+    ec2_type: str
+    monthly_price: float
+
+    # Return the type and state of the AMI
+    @staticmethod
+    def to_string():
+        return 'ami', 'available'
+
+    @staticmethod
+    def to_header() -> [str]:
+        return ['AMI ID',
+                'Name',
+                'State',
+                'Terminate After',
+                'Contact',
+                'Monthly Price',
+                'Region Name',
+                'AMI Type',
+                'OS'
+                'IOPS',
+                'Throughput']
+
+    def to_list(self) -> [str]:
+        return [self.resource_id,
+                self.name,
+                self.state,
+                self.terminate_after,
+                self.contact,
+                self.monthly_price,
+                self.region_name,
+                self.resource_type,
+                self.operating_system,
+                self.iops,
```

Comment:
storage and iops are only available on some amis, but I just implemented a function that assigns iops and volume_type if it exists in the block_device_mappings, otherwise iops and volume_type will just be None.